packages/react-components/src/organisms/Filter.tsx

```diff
@@ -54,26 +55,23 @@ const showMenuStyles = css({
 interface FilterProps<V extends string> {
   readonly filters?: Set<V>;
   readonly onChangeFilter?: (filter: V) => void;
-  readonly filterOptions: Option<V>[];
-  readonly filterTitle: string;
+  readonly filterOptions: ReadonlyArray<Option<V> | Title>;
 }
 export default function Filter<V extends string>({
   filters = new Set(),
   onChangeFilter = noop,
   filterOptions,
-  filterTitle,
 }: FilterProps<V>): ReturnType<React.FC> {
   const [menuShown, setMenuShown] = useState(false);
   useEffect(() => {
     setMenuShown(false);
-  }, [filterTitle]);
+  }, [filterOptions]);
```

Comment:
Not sure this actually needs anything. I think on mount we just want to ensure the menu is closed
```suggestion
  }, []);
```